src/js/angular/controllers.js

```diff
@@ -23,6 +23,7 @@ import './guides/directives';
 import {GUIDE_PAUSE} from './guides/tour-lib-services/shepherd.service';
 import 'angular-pageslide-directive/dist/angular-pageslide-directive';
 import 'angularjs-slider/dist/rzslider.min';
+import _ from "lodash";
```

Comment:
import only the debounce function and not the entire thing please

Reply:
Done :smile: